pkgs/development/python-modules/pytest-console-scripts/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchpatch
+, fetchPypi
+, pytestCheckHook
+, python
+, mock
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-console-scripts";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c1344853e80d8096403e36b270254a094071d0c4f1533e94a91d7386f0f1ccfd";
+  };
+  patches = [
+    # Handle easy_install missing during tests.
+    (fetchpatch {
+      url = "https://github.com/kvas-it/pytest-console-scripts/commit/d7dbca3ee48bd980f8213b77db16256e753740cb.patch";
+      sha256 = "sha256-EcKucRSQpWbYw6oV4cFOYMCiOaYAGyAq8lfS89R06g8=";
+    })
+  ];
+  postPatch = ''
+    # TODO: should we upstream this one?
+    substituteInPlace pytest_console_scripts.py --replace "['python']" "[sys.executable]"
+
+    substituteInPlace tests/test_run_scripts.py --replace "#!/usr/bin/env python" "#!${python.interpreter}"
```

Comment:
Use `patchShebangs`.

Reply:
`patchShebangs` doesn't seem applicable here: the shebang is not the file's interpreter, but instead is written to a script generated within.